### ISSUE TITLE
Inconsistent language: "token" vs "one-time authentication code"

### ISSUE THREAD

**Issue by neynah** [sandstorm-io/sandstorm#2549]
The user noted some confusion and said it took them a moment to guess that they were the same thing.

![image](https://cloud.githubusercontent.com/assets/855341/18523575/b0f4f192-7a69-11e6-82d6-db4b723f3f3a.png)

**Comment by paulproteus** [comment]
I agree that this is confusing. @neynah can you submit a mockup of what we should do instead? You pick, and then we implement. :)

**Comment by zarvox** [comment]
"To minimize confusion, we should remove the `Enter your token:` form entirely and have people rely on the link from their email."

(only half joking)

**Comment by paulproteus** [comment]
I think @zarvox 's proposal is the best one I've heard so far on this ticket. :)
